recipes/pixman/all/test_v1_package/conanfile.py

```diff
@@ -0,0 +1,17 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake", "cmake_find_package"
```

Comment:
```suggestion
    generators = "cmake", "cmake_find_package_multi"
```
Let's try with cmake_find_package_multi generator instead. It's based on the build type.